docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -60,8 +60,7 @@ teleport:
   #
   # We recommend to use tools like `pwgen` to generate sufficiently random
   # tokens of 32+ byte length.
-  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
-  auth_server: 10.1.1.10:3025
+  auth_server: 10.1.1.10:443
```

Comment:
```suggestion
  proxy_server: 10.1.1.10:443
```

With config version `v3` you must use `auth_server` for auth servers and `proxy_server` for proxy servers.